docs/source/sctool/backup.rst

```diff
@@ -476,8 +121,6 @@ backup files parameters
 
```

Comment:
@mmatczuk I couldn't find a YAML file for the command ``backup files``. Should we delete it from the doc (rst)?